test/chain/chain_test.dart

```diff
@@ -14,13 +14,13 @@ typedef ChainErrorCallback = void Function(dynamic stack, Chain chain);
 
 void main() {
   group('Chain.parse()', () {
-    test('parses a real Chain', () {
-      return captureFuture(() => inMicrotask(() => throw 'error'))
-          .then((chain) {
-        expect(
-            Chain.parse(chain.toString()).toString(), equals(chain.toString()));
-      });
-    });
+    test(
+        'parses a real Chain',
+        () =>
```

Comment:
I'm skeptical about enabling this rule blindly... The reduced nesting version is easier to read. `test` arguments should never be `() =>` IMO.

We could maybe keep the lint and rewrite the tests to avoid return?

```dart
test('parses  a real Chain', () async {
  await captureFuture(...);
});
```

Reply:
fixed!